bin/ssh_scan

```diff
@@ -127,6 +126,12 @@ scan") do |file|
   opts.on("-P", "--policy [FILE]",
           "Custom policy file (Default: Mozilla Modern)") do |policy|
     options["policy"] = policy
+    unless File.exist?(options["policy"])
+      puts opt_parser.help
+      puts "\nReason: policy file supplied is not a file"
+      exit 1
+    end
+    options["policy_file"] = SSHScan::Policy.from_file(options["policy"])
```

Comment:
From what I understand, the user doesn't need to always specify a policy file. But when he/she doesn't, the policy file is found on a first match basis, failing which we use the fallback file.